routing_model/src/flows_dst.rs

```diff
@@ -346,3 +346,58 @@ impl CheckAndProcessElderChange {
         self.clone()
     }
 }
+
+#[derive(Debug, PartialEq, Clone)]
+pub struct CheckOnlineOffline(pub State);
```

Comment:
make sense with CheckAndProcessElderChange. I guess they could be moved in their own file in the future (not particularly a dst thing